packages/web-react/tests/providerTests/requiredPropsTest.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import React, { ComponentType } from 'react';
+
+export const requiredPropsTest = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Component: ComponentType<any>,
+  role: string,
+  attribute: string,
+  attributeValue: string,
+) => {
+  it(`should have correct ${attribute}`, () => {
+    render(React.cloneElement(<Component />, { [attribute]: attributeValue }));
```

Comment:
```suggestion
    render(cloneElement(<Component />, { [attribute]: attributeValue }));
```

Please, import functions from the module instead of using a React object. This will be deprecated in the future version and also to be consistent with the rest of the code. We do this this way in all of our codebase. Thanks :-)